doc/classes/Vector3.xml

```diff
@@ -251,11 +251,13 @@
 			<return type="Vector3" />
 			<param index="0" name="uv" type="Vector2" />
 			<description>
+				Returns the [Vector3] from an octahedral-compressed form created using [method octahedron_encode] (stored as a [Vector2]).
 			</description>
 		</method>
 		<method name="octahedron_encode" qualifiers="const">
 			<return type="Vector2" />
 			<description>
+				Returns the octahedral-encoded form of this [Vector3] as a [Vector2]. This form of [i]lossy[/i] compression can be used to pass greater amounts of data without increasing storage or memory requirements. See also [method octahedron_decode].
```

Comment:
Not sure this part is correct.

Using this method both ways doesn't quite return the original value:

```gdscript
print(Vector3.octahedron_decode(Vector3(0, 0.5, 1).octahedron_encode()))
```

Gives me:

```
(0, 0.447214, 0.894427)
```

The value also appears to always be normalized, but I guess this is intended.